cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -152,15 +160,61 @@ func Run(gadgetRoot, device string, options Options) error {
 	// store the encryption key as the last part of the process to reduce the
 	// possiblity of exiting with an error after the TPM provisioning
 	if options.Encrypt {
-		if err := rkey.Store(options.RecoveryKeyFile); err != nil {
-			return fmt.Errorf("cannot store recovery key: %v", err)
+		if err := tpmSealKey(key, rkey, options); err != nil {
+			return fmt.Errorf("cannot seal the encryption key: %v", err)
 		}
+	}
+
+	return nil
+}
+
+// TODO:UC20: get cmdline definition from bootloaders
+var kernelCmdlines = []string{
+	// run mode
+	"console=ttyS0 console=tty1 panic=-1 systemd.gpt_auto=0 rd.systemd.unit=basic.target snapd_recovery_mode=run",
+}
+
+func tpmSealKey(key partition.EncryptionKey, rkey partition.RecoveryKey, options Options) error {
+	if err := rkey.Store(options.RecoveryKeyFile); err != nil {
```

Comment:
in a follow up we should rename all methods that write to files called Store(...) to either Save or Write.  Store in snapd context reminds too much of store the noun